File: server/src/main/scala/kpn/api/common/node/NodeReferences.scala
package kpn.api.common.node

case class NodeReferences(
  networkReferences: Seq[NodeNetworkReference] = Seq.empty,
  routeReferences: Seq[NodeOrphanRouteReference] = Seq.empty
) {
  def isEmpty: Boolean = networkReferences.isEmpty && routeReferences.isEmpty
}

File: backend/api/src/main/java/kpn/api/common/node/NodeReferences.java
package kpn.api.common.node;

import kpn.api.common.common.Reference;

import com.google.common.collect.ImmutableList;

public record NodeReferences(
  ImmutableList<Reference> networkReferences,
  ImmutableList<Reference> routeReferences
) {
}
